health-check-tcp/test.yaml

```diff
@@ -0,0 +1,20 @@
+# Copyright 2021 Google LLC
```

Comment:
Please delete this file. We should only skip testing when necessary.

Reply:
Done!